src/components/backdropScreensaver/plugin.js

```diff
@@ -25,12 +21,12 @@ define(['connectionManager'], function (connectionManager) {
                 Limit: 200
             };
 
-            var apiClient = connectionManager.currentApiClient();
-            apiClient.getItems(apiClient.getCurrentUserId(), query).then(function (result) {
+            const apiClient = connectionManager.currentApiClient();
+            apiClient.getItems(apiClient.getCurrentUserId(), query).then((result) => {
 
                 if (result.Items.length) {
 
-                    require(['slideshow'], function (slideshow) {
+                    import('slideshow').then(slideshow => {
```

Comment:
`TypeError: slideshow is not a constructor`
`slideshow = {default: ƒ}`

Reply:
We should probably note somewhere that babel does this by default